README.md

```diff
@@ -1,17 +1,12 @@
-# Raspberry Pi RP2350 Pico SDK Examples - Early Access
+# Raspberry Pi Pico SDK Examples
 
-## RP2350 Instructions
-
-Everything below this section is from the stock pico-examples, so ignore URLs etc., but generally instructions are the same.
-
-The Pico SDK default continues to be to build for RP2040 (PICO_PLATFORM=rp2040), so to build for RP2350, you need to pass
+By default, the Pico SDK targets builds for RP2040 (PICO_PLATFORM=rp2040). To build for RP2350 instead, pass
```

Comment:
Is it worth mentioning that you can also use e.g. `-DPICO_BOARD=pico2` or `-DPICO_BOARD=pico2_w` and that will "automagically" select RP2350 for you?